interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+			<label for="GU1"> <?php echo xlt('Hematura');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU2" name="GU2" type="checkbox" value="on" >
+			<label for="GU2"> <?php echo xlt('Change in Frequency of Urination');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU3" name="GU3" type="checkbox" value="on" >
+			<label for="GU3"> <?php echo xlt('Urinary Urgency');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU4" name="GU4" type="checkbox" value="on" >
+			<label for="GU4"> <?php echo xlt('Incontinence');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU5" name="GU5" type="checkbox" value="on" >
+			<label for="GU5"> <?php echo xlt('Slow Urination');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU6" name="GU6" type="checkbox" value="on" >
+			<label for="GU6"> <?php echo xlt('Irregular Periods');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU7" name="GU7" type="checkbox" value="on" >
+			<label for="GU7"> <?php echo xlt('Hot Flashes');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GU8" name="GU8" type="checkbox" value="on" >
+			<label for="GU8"> <?php echo xlt('Post Menopausal Bleeding');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="GU_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>
+
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('MSK:');?></B>
+	</TD>
+	</TR>
+	
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="MSK1" name="MSK1" type="checkbox" value="on" >
+			<label for="MSK1"> <?php echo xlt('Back Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK2" name="MSK2" type="checkbox" value="on" >
+			<label for="MSK2"> <?php echo xlt('Muscle Aches');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK3" name="MSK3" type="checkbox" value="on" >
+			<label for="MSK3"> <?php echo xlt('Arthralgias');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK4" name="MSK4" type="checkbox" value="on" >
+			<label for="MSK4"> <?php echo xlt('Muscle Cramps');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK5" name="MSK5" type="checkbox" value="on" >
+			<label for="MSK5"> <?php echo xlt('Localized Joint Pain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK6" name="MSK6" type="checkbox" value="on" >
+			<label for="MSK6"> <?php echo xlt('Localized Joint Swelling');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK7" name="MSK7" type="checkbox" value="on" >
+			<label for="MSK7"> <?php echo xlt('Localized Joint Stiffness');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation